src/AutoBogus/Generators/VersionGenerator.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace AutoBogus.Generators
+{
+  internal sealed class VersionGenerator
+    : IAutoGenerator
+  {
+    object IAutoGenerator.Generate(AutoGenerateContext context)
+    {
+      return new Version(
+        context.Faker.Random.Int(min: 0),
```

Comment:
Just wondering if it makes sense to use `Faker.System.Version()` here?

Reply:
Good catch! I'm not familiar with Faker and didn't know that that existed. I can't think of any way that this makes more sense than that :-)

Reply:
Apologies for the delayed contribution. As nnnvvvcv points out, Bogus does have a handler for generating `Version` instances. Also, if you use the `AutoBogus.Conventions` package, it will auto -wire instances of `Version`. 

You can see the implementation at the below and usage guide in the main AutoBogus readme.

https://github.com/nickdodd79/AutoBogus/blob/8633953e1f129797180a848ee1f7a43c06a2be0c/src/AutoBogus.Conventions/ConventionByNameGenerators.cs#L2164

Reply:
Ahh okay :-) Then I guess this PR isn't needed!